recipes/chunkio/all/conanfile.py

```diff
@@ -10,7 +11,7 @@ class ChunkIOConan(ConanFile):
     description = "Simple library to manage chunks of data in memory and file system"
     license = "Apache-2.0"
     url = "https://github.com/conan-io/conan-center-index"
-    homepage = "https://github.com/edsiper/chunkio"
+    homepage = "https://github.com/fluent/chunkio"
```

Comment:
👍 - user changed usarname, github is redirecting to the new one, great catch @toge!